src/20_Components/amp-analytics.html

```diff
@@ -341,6 +346,69 @@
   }
   </script>
   </amp-analytics>
-
+  <!-- ## Form -->
+  <!-- You can track events triggered by `amp-form`. Find a list of supported event [here](https://www.ampproject.org/docs/reference/components/amp-form#analytics-triggers)-->
+  <div>
+    <amp-analytics>
+      <script type="application/json">
+        {
+          "requests": {
+            "event": "https://amp-publisher-samples-staging.herokuapp.com/amp-analytics/ping?user=[[.]]&account=ampbyexample&event=${eventId}"
+          },
+          "triggers": {
+            "formSubmit": {
+              "on": "amp-form-submit",
+              "request": "event",
+              "selector":"#testForm",
+              "vars": {
+                "eventId": "form-submit"
+              }
+            },
+            "formSubmitSuccess": {
+              "on": "amp-form-submit-success",
+              "request": "event",
+              "selector":"#testForm",
+              "vars": {
+                "eventId": "form-submit-success"
+              }
+            },
+            "formSubmitError": {
+              "on": "amp-form-submit-error",
+              "request": "event",
+              "selector":"#testForm",
+              "vars": {
+                "eventId": "form-submit-error"
+              }
+            }
+          }
+        }
+      </script>
+    </amp-analytics>
+    <form 
+        id="testForm"
+        method="post"
+        class="p2"
+        action-xhr="/components/amp-form/submit-form-input-text-xhr"
+        target="_top">
+    <div class="ampstart-input inline-block relative m0 p0 mb3">
+      <input type="text"
+          class="block border-none p0 m0"
+          name="name"
+          placeholder="Name..."
+          required>
+    </div>
+    <input type="submit" value="Subscribe" class="ampstart-btn caps">
+    <div submit-success>
+      <template type="amp-mustache">
```

Comment:
I think you can simplify this by removing the templates (then you can also remove the amp-mustache import)

Reply:
done